MobileApplication/MobileApplication/Views/AboutPage.xaml.cs

```diff
@@ -21,6 +21,10 @@ public partial class AboutPage : ContentPage
     {
         private NewCategoryPage _newCategory;
         public CategoryPage categoryPageNew;
+        private float b;
+        private float income;
+        private float expenses;
```

Comment:
Why its float if all our transaction amounts is in decimal?

Reply:
I changed it